tests/test_ops.py

```diff
@@ -124,3 +126,98 @@ def test_to_ops_stencil(self, _accesses):
             Symbol(stencil_name),
             Literal('"%s"' % stencil_name.upper())
         )
+
+    def test_create_ops_dat_time_function(self):
+        grid = Grid(shape=(4))
+
+        u = TimeFunction(name='u', grid=grid, space_order=2)
```

Comment:
The test should work with and without `save` and as pointed out above, with something like `save=200` (which is quite small) this is gonna blow up.